Classes/Issues/Comments/Reactions/IssueCommentReactionCell.swift

```diff
@@ -156,6 +156,23 @@ UICollectionViewDelegateFlowLayout {
         let model = reactions[indexPath.item]
         cell.label.text = "\(model.content.emoji) \(model.count)"
         cell.contentView.backgroundColor = model.viewerDidReact ? Styles.Colors.Blue.light.color : .clear
+        
+        var users = model.users
+        guard users.count > 0 else { return cell }
+        
+        let difference = model.count - users.count
+        if difference > 0 {
+            let plural = difference > 1 ? "s" : ""
+            users.append("\(difference) other\(plural)")
+        }
+        
+        let lastUser = users.removeLast()
+        var message = users.joined(separator: ", ")
+        message += users.count > 0 ? " and " + lastUser : lastUser
+        message += " reacted with " + model.content.localizedString + " emoji"
```

Comment:
A few things: 

- these aren't localized strings 
- we need to use localized formatting strings and not con at (other languages comma separate differently)

So probably need to check which case it is (ither count or not) and then use a formatting string like `NSLocalizedString("%@ and %i others", "").

Also let's drop the "with heart", the tooltip already displays above the emoji, should be obvious.